### PR TITLE
Fix small comments bugs [SCI-3556]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-3556](https://biosistemika.atlassian.net/browse/SCI-3556)

### What was done
Fix multiple comments bugs

### CODE REVIEW

**Comment by keonjeo** [comment]
Hi @aignatov-bio  can you take some screenshot for your pr. It will make other guys more clearly to know what you change. lol

File: app/views/shared/comments/_comments.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="comments-container" data-object-id = <%= object.id %>> 
   <% per_page = Constants::COMMENTS_SEARCH_LIMIT %>
-  <div class="content-comments">
+  <div class="content-comments inline_scroll_block">
```
Comment (keonjeo) [inline app/views/shared/comments/_comments.html.erb]: where do you define this `inline_scroll_block` class name? btw, can we use `inline-scroll-block` name?

**Comment by aignatov-bio** [comment]
@keonjeo - this is small visual fixes. Nothing interesting.

**Comment by keonjeo** [comment]
> @keonjeo - this is small visual fixes. Nothing interesting.

well. Thanks.  let it go. lol

**Pull request merged**